OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,25 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.33
+
+## Backup Service
+* Encryption key collision during restore is now prevented and logged. Keys from system tables will no longer be included when restoring. (OPSC-17115)
+
+## Core
+* Added gossip configuration files to diagnostics tarball. (OPSC-17480)
+* Added {{alert_on_no_host_available}} to the {{[cassandra]}} section of the cluster conf. When set to true an alert will be triggered if opscenterd is unable to make a cql connection to the monitored cluster. (OPSC-17398)
+
+## CVE
+* Added {{redirect_host}} config value to the {{[webserver]}} section of {{opscenterd.conf}} to mitigate host header injection vulnerability. (OPSC-17265)
```

Comment:
This provides too much information allowing a potential attacker target older versions of OpsCenter. Also, it doesn't address a specific CVE (even though it's a security issue).
I propose to rewrite this under Core, stating that we added parameter `redirect_host` to force the hostname used in redirects - or something along those lines.